components/channel_header_dropdown/mobile_channel_header_dropdown.tsx

```diff
@@ -68,7 +70,7 @@ export default class MobileChannelHeaderDropdown extends React.PureComponent {
                         {(title) => (
                             <span
                                 className='fa fa-angle-down header-dropdown__icon'
-                                title={title}
+                                title={`${title}`}
```

Comment:
why did you change it, the current version doesn’t change anything
```suggestion 
                                title={title}
```

Reply:
Same as I mentioned about ariaLabel.